sound/soc/codecs/rt711-sdca.c

```diff
@@ -1330,6 +1371,8 @@ int rt711_sdca_init(struct device *dev, struct regmap *regmap,
 	 */
 	rt711->hw_init = false;
 	rt711->first_hw_init = false;
+	rt711->fu0f_dapm_mute = rt711->fu0f_mixer_mute = true;
+	rt711->fu1e_dapm_mute = rt711->fu1e_mixer_mute = true;
```

Comment:
Should we read the mute status from register instead of assuming it be muted?

Reply:
> Should we read the mute status from register instead of assuming it be muted?

The register setting is mute in default.